gufe/tests/test_proteincomponent.py

```diff
@@ -359,6 +360,12 @@ def test_pdb_ion_invalid(self, custom_pdb_ion):
         with pytest.raises(ValueError, match="ab7 in residue CL at index 1."):
             self.cls.from_pdb_file(next(pdb_generator))
 
+    def test_protein_valence_error(self):
```

Comment:
I'm just curious, without the code fix above, what happens when you run this test? I'm not at my computer or I would do this myself.